src/dependencies.jl

```diff
@@ -864,15 +864,15 @@ macro install(_libmaps...)
                         depsfile_buffer = IOBuffer()
                         println(depsfile_buffer,
                             """
-                            # This is an auto-generated file; do not edit
+                            # This is an auto-generated file; do not edit this file.
```

Comment:
not when BinDeps updates, but when they update. see below where the content of deps.jl is compared.